pkg/k8shandler/kibana/reconciler.go

```diff
@@ -363,6 +364,12 @@ func isDeploymentDifferent(current *apps.Deployment, desired *apps.Deployment) (
 		different = true
 	}
 
+	if current.Spec.Replicas != desired.Spec.Replicas {
```

Comment:
i think we need to dereference this to check the values: 
*current.Spec.Replicas != *desired.Spec.Replicas

https://godoc.org/k8s.io/api/apps/v1#DeploymentSpec

Reply:
Fxied